.env

```diff
@@ -9,3 +9,11 @@ REACT_APP_DISABLE_SENTRY=0
 REACT_APP_FEATURE_PUBLIC_HANKKEET=1
 REACT_APP_FEATURE_HANKE=1
 REACT_APP_FEATURE_ACCESS_RIGHTS=0
+REACT_APP_WARNING_NOTIFICATION_LABEL=""
+REACT_APP_WARNING_NOTIFICATION_TEXT_FI=""
+REACT_APP_WARNING_NOTIFICATION_TEXT_SV=""
+REACT_APP_WARNING_NOTIFICATION_TEXT_EN=""
+REACT_APP_ERROR_NOTIFICATION_LABEL=""
+REACT_APP_ERROR_NOTIFICATION_TEXT_FI=""
+REACT_APP_ERROR_NOTIFICATION_TEXT_SV=""
+REACT_APP_ERROR_NOTIFICATION_TEXT_EN=""
```

Comment:
I would save the REACT_APP prefix for React (and addons) functionality. Another prefix would be better for stuff we have created ourselves and are free to rename how we like. Something like `HAITATON_UI`

Reply:
That would be good, but with Create React App, custom environment variables have to have REACT_APP prefix. In the documentation it's said: `Note: You must create custom environment variables beginning with REACT_APP_. Any other variables except NODE_ENV will be ignored to avoid accidentally exposing a private key on the machine that could have the same name.`

Reply:
I tested that and it seems to apply to us also.